README.md

```diff
@@ -22,15 +22,13 @@ To get started, run the following commands:
 
 `git clone https://github.com/wizeline/serverless-fullstack`
 `cd serverless-fullstack`
-`npm i`
-`npm setup --appName="Awesome App" --shortName=awesomeapp`
-# TODO: We need to update setup script to add credentials/profiles to the aws credentials file for each stage (if they only provide dev creds, we'll create the staging and prod profiles using the dev creds) npm run deploy
-
-Replace "Awesome App" with the full name of your application. Optionally, provide a shortName that will be used for the AWS CloudFormation Stack name. If shortName isn't provided, it will be constructed based on applicationName.
+`cp example.setup.config.json setup.config.json`
```

Comment:
Let's just create the file as `setup.config.json` instead of having them cp `example.setup.config.json`.